tests/tests_integration/test_api/test_simulators/test_models.py

```diff
@@ -278,3 +281,50 @@ def test_update_model(self, cognite_client: CogniteClient, seed_resource_names)
         assert model_updated.description == "updated description"
         assert model_updated.name == "updated name"
         cognite_client.simulators.models.delete(external_ids=[model_updated.external_id])
+
+    def test_create_model_and_revisions_with_data(
+        self, cognite_client: CogniteClient, seed_resource_names: ResourceNames
+    ) -> None:
+        models = cognite_client.simulators.models.list(
+            simulator_external_ids=seed_resource_names.simulator_external_id, limit=1
+        )
+
+        model_revisions = cognite_client.simulators.models.revisions.list(
+            limit=1, model_external_ids=models[0].external_id
+        )
+
+        model_revision = model_revisions[0]
+
+        model_revision_data_list = cognite_client.simulators.models.revisions.data.list(
+            model_revision_external_id=model_revision.external_id
+        )
+
+        revision_data = cognite_client.simulators._post(
+            "/simulators/models/revisions/data/update",
+            headers={"cdf-version": "alpha"},
+            json={
+                "items": [
+                    {
+                        "modelRevisionExternalId": model_revision.external_id,
+                        "update": {"flowsheets": {"set": SIMULATOR_MODEL_REVISION_DATA_FLOWSHEET}},
+                    }
+                ]
+            },
+        )
+
+        assert revision_data.status_code == 200
+
+        model_revision_data = model_revision.get_data()
+        model_revision_data_list = cognite_client.simulators.models.revisions.data.list(
```

Comment:
```suggestion
        model_revision_data_list = cognite_client.simulators.models.revisions.get_data(
```
i think we should expose it through the revisions API, it comes tother